### PR TITLE
Test that the section title does not contain HTML markup

### PR DESCRIPTION
**Pull request by Dantemss**
https://github.com/openstax/cnx-automation/issues/146

### CODE REVIEW

File: tests/webview/ui/test_content.py [hunk 1]
```diff
@@ -163,6 +163,22 @@ def test_get_this_book(webview_base_url, selenium):
     assert offline_zip_displayed == downloads.is_offline_zip_available
 
 
+@markers.webview
```
Comment (m1yag1) [inline tests/webview/ui/test_content.py]: Add `@markers.test_case()`

File: tests/webview/ui/test_content.py [hunk 2]
```diff
@@ -163,6 +163,22 @@ def test_get_this_book(webview_base_url, selenium):
...
+@markers.nondestructive
+def test_section_title(webview_base_url, selenium):
```
Comment (m1yag1) [inline tests/webview/ui/test_content.py]: Be more specific with the test name. `test_section_title_for_no_markup`

File: tests/webview/ui/test_content.py [hunk 3]
```diff
@@ -163,6 +163,22 @@ def test_get_this_book(webview_base_url, selenium):
...
+
+    # THEN the section title does not contain HTML markup
+    section_title = content.section_title
+    assert '<' not in section_title
+    assert '>' not in section_title
+
+
 @markers.webview
 @markers.nondestructive
 def test_content(webview_base_url, selenium):
```
Comment (m1yag1) [inline tests/webview/ui/test_content.py]: This also looks like it needs a more specific name and testrail marker

Comment (m1yag1) [reply tests/webview/ui/test_content.py]: oh snap, that's from the old import where i didn't include that, my badz. @kerwinso can you help dante locate the case id for this one?

File: tests/webview/ui/test_content.py (hunk 1 above)
Comment (Dantemss) [reply tests/webview/ui/test_content.py]: Added

File: tests/webview/ui/test_content.py (hunk 2 above)
Comment (Dantemss) [reply tests/webview/ui/test_content.py]: Changed the test name to `test_section_title_for_no_markup`

**Pull request merged**